ocs_ci/deployment/deployment.py

```diff
@@ -398,7 +398,7 @@ def do_deploy_ocs(self):
                 # if we have Globalnet enabled in case of submariner with RDR
                 # we need to add a flag to storagecluster
                 if (
-                    config.ENV_DATA.get("enable_globalnet", True)
+                    get_primary_cluster_config().ENV_DATA.get("enable_globalnet", True)
                     and config.MULTICLUSTER["multicluster_mode"] == "regional-dr"
```

Comment:
```suggestion
                    config.MULTICLUSTER["multicluster_mode"] == "regional-dr" and 
                    get_primary_cluster_config().ENV_DATA.get("enable_globalnet", True) 
```
I think it will be better first check if it's regional-dr and then try to get primari cluster config

Reply:
sure will do that same

Reply:
Done